### ISSUE TITLE
Consider adding renv() for reproducible environments

### ISSUE THREAD

**Issue by cnolanminich** [philiporlando/dagster-and-r#3]
currently this project uses `poetry` for python packages, but nothing for the R packages -- [renv](https://rstudio.github.io/renv/articles/renv.html) is a great solution for this to make this project more reproducible.

**Comment by philiporlando** [comment]
I agree that this project could benefit from incorporating `{renv}`. Since this project was published, I've made more of an effort to establish reproducible R environments in [other projects](https://github.com/philiporlando/docker-duckdb-r).
